src/codegen/templates/_task_modules/ai_channel_collection.py.mako

```diff
@@ -37,8 +37,8 @@ class AIChannelCollection(ChannelCollection):
             name_to_assign_to_channel (Optional[str]): Specifies a name to
                 assign to the virtual channel this method creates.
         Returns:
-            nidaqmx._task_modules.channels.ai_channel.AIChannel: 
-            
+            nidaqmx._task_modules.channels.ai_channel.AIChannel:
+
             Specifies the newly created AIChannel object.
```

Comment:
Adding a blank line between the return type and return description looks wrong. I guess it's moving the 1st line to the 2nd paragraph. I know some of the other files in this repo do this too but that doesn't make it better than what was there before.

https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html 

BTW this method is private so it doesn't really matter how we document it.

Reply:
The blank line between return type and description has been there in current file ([ai_channel_collection.py.mako](https://github.com/ni/nidaqmx-python/blob/205dc3bcd6f5c32c7d4f1ed46e66af30af65f927/src/codegen/templates/_task_modules/ai_channel_collection.py.mako#L41)). Based on the diff-view on generated ai_channel_collection.py, I believe it's my code editor removed whitespaces at line 41 on mako file when I saved the file after reverting the changes from first commit.
![image](https://github.com/ni/nidaqmx-python/assets/27721199/e46ddb2c-6b32-4ae4-9b33-7eb4385cb16c)

I find that other public methods having a blank line between return type and description, maybe I should keep the blank line to align with existing docstring format.

Reply:
Oh, I see. Removing trailing whitespace without changing the formatting is fine.